src/reformatters/common/dynamical_dataset.py

```diff
@@ -41,6 +47,30 @@ def reformat_kubernetes(self) -> None:
         """Run dataset reformatting using Kubernetes index jobs."""
         pass
 
+    def reformat_operational_update(self) -> None:
+        """Update an existing dataset with the latest data."""
+        final_store = self._final_store()
+        tmp_store = self._tmp_store()
+
+        jobs, template_ds = self.region_job_class.operational_update_jobs(
+            final_store=final_store,
+            tmp_store=tmp_store,
+            get_template_fn=self._get_template,
+            append_dim=self.template_config.append_dim,
+            all_data_vars=self.template_config.data_vars,
+            kubernetes_job_name="operational-update",
```

Comment:
A bit of a nit, but I wonder if these should be constants or enums  etc since they are checked explicitly and not arbitrary names.